Bump Sentry JavaScript to 7.51+ for Angular 16 support

### Environment

Angular 15 (want to move to 16) with Ionic 6

### Desired Result

Update peerDependencies and dependencies to at least 7.51.0 to enable compatibility with the current version of Angular 16.
https://github.com/getsentry/sentry-javascript/releases/tag/7.51.0

### Actual Result

npm ERR! This version of Sentry Capacitor is incompatible with the following installed packages:
npm ERR! @sentry/node version 7.51.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,7 +4,7 @@
 
 ### Dependencies
 
-- Bump Sentry JavaScript SDK to `7.54.0` ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
+- Bump Sentry JavaScript SDK to `7.54.0` ([#389](https://github.com/getsentry/sentry-capacitor/pull/389))
   - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.54.0)
   - [diff](https://github.com/getsentry/sentry-javascript/compare/7.50.0...7.54.0)
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Bump Sentry JavaScript SDK to `7.54.0` ([#???](https://github.com/getsentry/sentry-capacitor/pull/???))
+  - [changelog](https://github.com/getsentry/sentry-javascript/releases/tag/7.54.0)
+  - [diff](https://github.com/getsentry/sentry-javascript/compare/7.50.0...7.54.0)
+
 ## 0.12.0
 
 ### Features

--- a/example/ionic-angular-v5/package.json
+++ b/example/ionic-angular-v5/package.json
@@ -14,9 +14,9 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular": "7.50.0",
+    "@sentry/angular": "7.54.0",
     "@sentry/capacitor": "file:.yalc/@sentry/capacitor",
-    "@sentry/replay": "7.50.0",
+    "@sentry/replay": "7.54.0",
     "rxjs": "~6.5.5",
     "tslib": "^2.0.0",
     "zone.js": "~0.11.4"

--- a/example/ionic-angular/package.json
+++ b/example/ionic-angular/package.json
@@ -14,9 +14,9 @@
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
     "@ionic/angular": "^5.0.0",
-    "@sentry/angular": "7.50.0",
+    "@sentry/angular": "7.54.0",
     "@sentry/capacitor": "file:.yalc/@sentry/capacitor",
-    "@sentry/replay": "7.50.0",
+    "@sentry/replay": "7.54.0",
     "rxjs": "~6.5.5",
     "tslib": "^2.0.0",
     "zone.js": "~0.11.4"

--- a/package.json
+++ b/package.json
@@ -38,9 +38,9 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.50.0",
-    "@sentry/react": "7.50.0",
-    "@sentry/vue": "7.50.0"
+    "@sentry/angular": "7.54.0",
+    "@sentry/react": "7.54.0",
+    "@sentry/vue": "7.54.0"
   },
   "peerDependenciesMeta": {
     "@sentry/angular": {
@@ -54,13 +54,13 @@
     }
   },
   "dependencies": {
-    "@sentry/browser": "7.50.0",
-    "@sentry/core": "7.50.0",
-    "@sentry/hub": "7.50.0",
-    "@sentry/integrations": "7.50.0",
-    "@sentry/tracing": "7.50.0",
-    "@sentry/types": "7.50.0",
-    "@sentry/utils": "7.50.0",
+    "@sentry/browser": "7.54.0",
+    "@sentry/core": "7.54.0",
+    "@sentry/hub": "7.54.0",
+    "@sentry/integrations": "7.54.0",
+    "@sentry/tracing": "7.54.0",
+    "@sentry/types": "7.54.0",
+    "@sentry/utils": "7.54.0",
     "@sentry/wizard": "^1.1.4",
     "promise": "^8.1.0"
   },
@@ -69,8 +69,8 @@
     "@capacitor/core": "^3.0.1 || ^4.0.0 || ^5.0.0",
     "@capacitor/ios": "^3.0.1 || ^4.0.0 || ^5.0.0",
     "@ionic/prettier-config": "^1.0.0",
-    "@sentry-internal/eslint-config-sdk": "7.50.0",
-    "@sentry-internal/eslint-plugin-sdk": "7.50.0",
+    "@sentry-internal/eslint-config-sdk": "7.54.0",
+    "@sentry-internal/eslint-plugin-sdk": "7.54.0",
     "@sentry/typescript": "5.20.1",
     "@types/jest": "^26.0.15",
     "eslint": "^7.13.0",

